src/components/charts/y-axis-label/y-axis-label.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Label } from 'recharts';
+
+const htmlToSvgSubscript = unitY => {
+  const splittedUnit = unitY.split('<sub>');
+  return splittedUnit.map(chain => {
+    if (chain.includes('</sub>')) {
+      const subChain = chain.split('</sub>');
+      return (
+        <tspan>
+          <tspan dy="4" fontSize="11px">{subChain[0]}</tspan>
+          <tspan dy="-4">{subChain[1]}</tspan>
+        </tspan>
+      );
+    }
+    return <tspan>{chain}</tspan>;
+  });
+};
+
+const yAxisLabel = unit => (
+  <Label
+    content={() => (
+      <text x="8" y="20" fontSize="13px" stroke="#b1b1c1" strokeWidth="0.5">
```

Comment:
It would be nice to get some className or theme prop here to customize the styles if needed

Reply:
It makes sense, already fixed, now it is possible to pass additional `className` and `x`, `y` position coordinates for the unit label, thanks :)